terraform/Makefile

```diff
@@ -54,12 +54,14 @@ lambda:
 	@echo "Zip and copy source files to dist/"
 	@rm -rf dist && mkdir dist
 	@find ./source -maxdepth 2  -name "node_modules" -exec rm -rf "{}" \;
-	@cd ./source/configure && npm install --production && zip -q -r9 ../../dist/marsha_configure.zip *;
-	@cd ./source/encode && npm install --production && zip -q -r9 ../../dist/marsha_encode.zip *;
-	@cd ./source/update_state && npm install --production && zip -q -r9 ../../dist/marsha_update_state.zip *;
+	@cd ./source/configure && yarn install --frozen-lockfile --production=true && zip -q -r9 ../../dist/marsha_configure.zip *;
+	@cd ./source/encode && yarn install --frozen-lockfile --production=true && zip -q -r9 ../../dist/marsha_encode.zip *;
+	@cd ./source/update_state && yarn install --frozen-lockfile --production=true && zip -q -r9 ../../dist/marsha_update_state.zip *;
 
 .PHONY: test
 test: ## test all lambda packages
 test:
 	@echo "Test all lambda packages"
-	@cd ./source/configure && npm install && npm test;
+	@cd ./source/configure && rm -rf node_modules && yarn install --frozen-lockfile && yarn test;
+	@cd ./source/encode && rm -rf node_modules && yarn install --frozen-lockfile && yarn test;
+	@cd ./source/update_state && rm -rf node_modules && yarn install --frozen-lockfile && yarn test;
```

Comment:
Instead of using a radical approach as `rm -rf node_modules`, using `yarn autoclean --prod` wouldn't be sufficient?

Reply:
I think autoclean is more intended to remove needless files inside dependencies, rather than remove dependencies folders? (https://yarnpkg.com/en/docs/cli/autoclean)

To get the behavior we need I don't know of any other way than deleting `node_modules`. Re-running the install should take less than ~2s as everything should already be in yarn's cache.

Reply:
Ok then.